tony-core/src/main/java/com/linkedin/tony/horovod/SlotInfo.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2021 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.tony.horovod;
+
+public class SlotInfo {
```

Comment:
would be useful to have some documentation for slotinfo here.. i know that is in Horovod documentation, but would be nice to have some info here.

Reply:
Got it.